.github/.wordlist.txt

```diff
@@ -1560,3 +1560,23 @@ zhengyaohan
 Zigbee
 zigbeealliance
 zigbeethread
+CommandInvalidInState
```

Comment:
These should perhaps be in backticks in the relevant .md so they don't need to be added here....

Reply:
Fixed [here](https://github.com/project-chip/connectedhomeip/pull/29159/commits/3ae6d2cc179fbf3cc6fa6b4741dee540c6cd443e).